backend/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM mcr.microsoft.com/dotnet/core/aspnet:3.1 AS base
 WORKDIR /app
-EXPOSE 80 443
+EXPOSE 443 8080
```

Comment:
Switch docker to use 8080 instead of 80 to be consistent with the api Openshift pod. This means that I don't have to test one setup on docker and a different setup on Openshift.

Reply:
Thanks for doing this. To align more with Openshift we should consider that in Openshift containers run as **non-root** so that means that docker can not expose "priviledged ports" (< 1024). Please consider removing the 443 port from here as well, or replace with with say 8443 (that's what I've seen done on other images)

Reply:
I would also be more inclined to make the PORT change via a supported env var within the Dockerfile itself to avoid having to hard-code the port (8080) into the source code - we should at least read it from configuration/environment on the NET Core side

See https://github.com/redhat-developer/s2i-dotnetcore/blob/master/3.1/runtime/Dockerfile#L38-L40 for how the Openshift images do it:

```Dockerfile

EXPOSE 8080

## By default, ASP.NET Core runs on port 5000. We configure it to match
## the container port.
ENV ASPNETCORE_URLS=http://*:8080

...
```